src/VbsEnclaveSDK/src/veil_enclave_lib/GenerateVeilExportStatements.ps1

```diff
@@ -0,0 +1,31 @@
+# Generates pragma linker statements for Veil Enclave exports,
+# using the 'Exports.cpp' file produced by edlcodegen.exe.
+param (
+    [string]$InputFile,
+    [string]$OutputFile
+)
+
+# Ensure input file exists
+if (-not (Test-Path $InputFile))
+{
+    Write-Error "Input file not found: $InputFile"
+    exit 1
+}
+
+@"
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Auto-generated export pragmas for the Veil Enclave static library.
+// This file should be included in the developer's enclave project to ensure
+// the Veil enclave export symbols are properly exposed by the enclave DLL.
+// Do not modify this file manually.
+#pragma once
+
+"@ | Out-File -FilePath $OutputFile
+
+# Extract pragma lines and append them to the output file.
+# example line: #pragma comment(linker, "/include:__AbiRegisterVtl0Callbacks_veil_abi__")
+Select-String -Path $InputFile -Pattern '#pragma comment\(linker,' |
+    ForEach-Object { $_.Line } |
+    Add-Content -Path $OutputFile
```

Comment:
super picky, but can we move this above line 15 and then save it to a variable (in mem) and then write the file all at once?

(then we are almost guaranteed to never have a half-baked file, and then we don't re-write the file with whtaever encoding Add-Content uses)